### ISSUE TITLE
Terraform doesn't identify DBs under subscription

### ISSUE THREAD

**Issue by GitaShmueli** [RedisLabs/terraform-provider-rediscloud#138]
Hi there,

Opening this issue on behalf of the customer.
Subscription number 1279165 managed by Terraform for a few months now. 
But there are some DBs that were created with the help of the Redis support team (List of DBs at the end of the request).
These DBs are not in the terraform state and are not even being shown as being deleted by terraform plan. 
They tried adding the config of one of the DBs to the terraform source code, but the plan shows as creating a new DB although in real there is already a DB running with the same name.

The instances are:
• remote-config-cache-general-prod-json
• remote-config-cache-stag-json
• remote-config-cache-sniper-prod-json
• remote-config-cache-suspects-prod-json
• remote-config-cache-tennis-prod-json
• remote-config-cache-zooba-prod-json

Basically Terraform is not seeing these DBs, but everything else in the subscription.
An API call returns all the DBs in the subscription successfully.

### Terraform Version
Provider RedisLabs/rediscloud version is 0.2.4.
Terraform version is 0.14.6.

### Affected Resource(s)
N/A

[###](url) Terraform Configuration Files
N/A

### Debug Output
N/A

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
Terraform should recognize all the DBs in the subscription.

### Actual Behavior
Terraform is trying to create a new DB although it exists with the same name.

### Steps to Reproduce
N/A

### Important Factoids
N/A

### References
N/A

**Comment by nir-redis** [comment]
This is by design. Once dbs are added in the UI, a terraform import operation is needed to update the terraform files to include the new dbs.
